instrument/EXED_Definition.xml

```diff
@@ -0,0 +1,590 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<instrument name="EXED" date="6th Mar 2017"
```

Comment:
I will probably pull the date field.  I'll look at the instrument definition and make sure it is more in line with standards.